eclass/cron.eclass

```diff
@@ -4,6 +4,7 @@
 # @ECLASS: cron.eclass
```

Comment:
This brought to mind the removal [bug](https://bugs.gentoo.org/693004), but I don't think it's too pressing

Reply:
hmm right. might as well just leave that one alone